<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inVelocidad">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>velocidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inDisancia">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>distancia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCorrer">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Correr</string>
    </property>
   </widget>
   <widget class="QLabel" name="runnerMan">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="corredor.qrc">:/new/corredor/assets/img.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="linea">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="corredor.qrc"/>
 </resources>
 <connections/>
</ui>
